quickwit-core/src/indexing/split.rs

```diff
@@ -171,6 +176,8 @@ impl Split {
                 );
             }
         };
+
+        self.metadata.size_in_bytes += doc_size as u64;
```

Comment:
Do we really want to have the size in bytes of the raw json docs? From a user that is looking at this metadata information on the split, I think this is misleading.
I would expect something like `SegmentSpaceUsage` given by `tantivy`.
What do you think? we can add an issue about this and leave the code as is.

Reply:
Yes, I think `SegmentSpaceUsage` is relevant for the split and its segments,  but I think having the overall size of data that was ingested is equally relevant and possibly displaying `doc_size`  vs split_size (composed of segment space usage) gives more send to the user

Reply:
mmm, but the doc_size can be very different from the split. I will open an issue to discuss that, this is minor.